docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,98 @@
+                    - look for the power validation points in power_validation_pts_dict: `if power_validation_pts_dict[[cwt, ect]]:`
+                        - we are expecting to see multiple validation points aligning with the expected_validation_plr.  Create a list of part load ratios that are given.  Later we'll compare this with the expected list to make sure that all points are given: `given_plrs = []`
+                        - look at each power validation point in the list: `for power_validation_point in power_validation_pts_dict[[cwt, ect]]:`
+                            - get the load: `load = power_validation_point.load`
+                            - get the given power: `given_power = power_validation_point.result`
+                            - calculate the PLR by dividing the load by the given capacity at these operating conditions: `plr = load / given_capacities[[cwt, ect]]`
+                            - check whether the plr is one of the plrs that we need to check: `if plr in expected_validation_plr:`
+                                - add the plr to the list of plrs provided: `given_plrs.append(plr)`
+                                - calculate eir_plr using the coefficients given: `eir_plr = plr_coefficients[0] + plr_coefficients[1] * plr + plr_coefficients[2] * plr^2`
+                                - calculate the eir_ft using the coefficients given: `eir_ft = eir_f_t_coefficients[0] + eir_f_t_coefficients[1] * cwt + eir_f_t_coefficients[2] * cwt^2 + eir_f_t_coefficients[3] * ect + eir_f_t_coefficients[4] * ect^2 + eir_f_t_coefficients[5] * cwt * ect
+                                - calculate the expected power using the formula:
+                                    - Chiller operating power = Rated Capacity × CAP-f-T × EIR-f-T × EIR-f-PLR × Chiller Input Power at Rated Conditions/Chiller Capacity at Rated Conditions
+                                - in this case, the given capacity under these operating conditions is the rated capacity * cap_ft, so the modified formula is: 
+                                    - Chiller operating power = given_capacity[[cwt, ect]] × EIR-f-T × EIR-f-PLR × Chiller Input Power at Rated Conditions/Chiller Capacity at Rated Conditions
+
+                                - `expected_power = given_capacity[[cwt, ect]] * eir_ft * eir_plr * rated_power/rated_capacity`
+                                - check whether the expected power and the given power are equal.  This should not be an exact match, but with a margin of error (see notes at the bottom for suggested margins of errors).  If the expected power and given power do not match, update power_validation_status and GO TO RULE ASSERTION: `if expected_power != given_power: power_validation_status = "Expected power does not match given power at CHWT: " + cwt + ", ECT: " + ect + " & " + plr".; GO TO RULE ASSERTION`
+                        - make sure that all of the expected plrs are covered by 
```

Comment:
I see this note but am not seeing where this is happening

Reply:
oops.  Old line that should've been deleted